docs/src/analysis/sensitivity.md

```diff
@@ -604,21 +604,22 @@ In this case we'd like to calculate the adjoint sensitivity of the scalar energy
 functional
 
 ```math
-G(u,p)=\int_{0}^{T}\frac{\sum_{i=1}^{n}u_{i}^{2}(t)}{2}dt
+G(u,p)=\int_{0}^{T}\frac{\sum_{i=1}^{3}u_{i}^{2}(t)}{2}dt
 ```
 
 which is
 
 ```julia
-g(u,p,t) = (sum(u).^2) ./ 2
+g(u,p,t) = (sum(u).^2) / 2
 ```
 
 Notice that the gradient of this function with respect to the state `u` is:
 
 ```julia
 function dg(out,u,p,t)
-  out[1]= u[1] + u[2]
-  out[2]= u[1] + u[2]
+  out[1]= u[1] 
+  out[2]= u[2]
+  out[3]= u[3]
```

Comment:
I believe that this section is for the Lotka Volterra model, so there are only two elements.

Reply:
This section is the gradient of the function g above and not regarding to any ODE.